Parser/pgen/keywordgen.py

```diff
@@ -0,0 +1,62 @@
+"""Generate Lib/keyword.py from the Grammar and Tokens files using pgen"""
+
+import argparse
+
+from .pgen import ParserGenerator
+
+TEMPLATE = '''\
```

Comment:
nitpick: You can use a raw-string to write \ rather than \\:

```
 TEMPLATE = r'''...'''
```

And use for example .lstrip() to remove the first newline before the shebang.

Reply:
Done in bffdb57